src/dbt_score/dbt_utils.py

```diff
@@ -16,13 +25,27 @@ class DbtLsException(Exception):
     """Raised when dbt ls fails."""
 
 
+def dbt_required(func: Callable[..., Any]) -> Callable[..., Any]:
+    """Decorator for methods that require dbt to be installed."""
+
+    def inner() -> None:
```

Comment:
We can use https://docs.python.org/3/library/functools.html#functools.wraps

to preserve docstrings, names etc of the original function 👌